### PR TITLE
feat: solved capacitance bugs

### LINKED ISSUE #1084: Bugs in capacitance measurement
**Actual Behaviour**

In the multimeter capacitor is not not working properly

**Expected Behaviour**

It should measure the capacitance

**Steps to reproduce it**

Make changes in the communication classes

**LogCat for the issue**

Not applicable

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by abhinavraj23**
Fixes #1084 

**Changes**: 
Device now able to measure capacitance

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: [Compress the `app-debug.apk` file into a `<feature>.rar` or `<feature>.zip` file and upload it here]
[capacitance.apk.zip](https://github.com/fossasia/pslab-android/files/2146947/capacitance.apk.zip)

### CODE REVIEW

**Comment by abhinavraj23** [comment]
@CloudyPadmal @harsh-2711 @Vikum94 @Avjeet Finally the the capacitor bugs are solved it is working, please test it in your devices and let know how in case of other bugs or how it can be further improvised.

File: app/src/main/java/org/fossasia/pslab/communication/ScienceLab.java [hunk 1]
```diff
@@ -2221,11 +2221,7 @@ public Double getCapacitance() {
                 return C;
             else if (V < GOOD_VOLTS[0] && V > 0.01 && CT < 40000) {
                 if (GOOD_VOLTS[0] / V > 1.1 && iterations < 10) {
-                    try {
-                        CT = CT * (int) GOOD_VOLTS[0] / (int) V;
-                    } catch (ArithmeticException e) {
-                        Log.e(TAG, "No capacitor connected or a short circuit!");
-                    }
+                        CT = (int) (CT * GOOD_VOLTS[0] / V);
```
[the remark this answers is not in the document]
Comment (harsh-2711) [reply app/src/main/java/org/fossasia/pslab/communication/ScienceLab.java]: @CloudyPadmal I don't think a try catch block is required as if V = 0 then it won't pass the if condition.

File: app/src/main/java/org/fossasia/pslab/communication/ScienceLab.java [hunk 2]
```diff
@@ -2253,14 +2249,17 @@ else if (CR == 3) {
                 mPacketHandler.sendByte((int) trim / 2);
             mPacketHandler.sendInt(chargeTime);
             Thread.sleep((long) (chargeTime * 1e-6 + .02));
-            int VCode = mPacketHandler.getInt();
+            int VCode;
+            do VCode = mPacketHandler.getVoltageSummation();
+            while (VCode == -1);
+            Log.v(TAG, "Voltage at function" + String.valueOf(VCode));
```
Comment (harsh-2711) [inline app/src/main/java/org/fossasia/pslab/communication/ScienceLab.java]: You can remove this too.

**Comment by abhinavraj23** [comment]
@harsh-2711 @Vikum94 Changes done :+1:

**Pull request merged**